src/command/FileCommands.cxx

```diff
@@ -233,3 +239,103 @@ handle_read_comments(Client &client, Request args, Response &r)
 
 	gcc_unreachable();
 }
+
+InputStreamPtr
+find_stream_art(const char *directory, Mutex &mutex, Cond &cond)
```

Comment:
Add Doxygen-style comments like the rest of the code does. "Obvious" things don't need to be documented (for some definition of "obvious"), but when it's about file names, you almost always need documentation, because the difference between filesystem charset and UTF-8 is important, and without explicit documentation, it's too easy to get wrong.
And if it's internal, then it should be `static`.